src/common/attributes/usePositionAttributes.js

```diff
@@ -108,6 +108,10 @@ export default (t) => useMemo(() => ({
     name: t('positionCoolantTemp'),
     type: 'number',
   },
+  engineTemp: {
+    name: t('positionEngineTemp'),
```

Comment:
I don't see this value in the strings. I also don't see it on the backend as a known attribute.

Reply:
https://github.com/traccar/traccar/blob/5e6c6119535f057e73d6b4fe3ba0e2617a7c537d/src/main/java/org/traccar/protocol/TeltonikaProtocolDecoder.java#L245

Reply:
We only add standard attributes.

What about the first part?

Reply:
It seems like my comment is not addressed.

Reply:
Should we change server side? What do you suggest for being able to see the engine temperature in the device status?

Reply:
If you want to add a new known attribute you should add it on the server side. The minimum requirement is that it is used by at least two different protocols. Then we can add the web app part.